Documentation: The example in the Read.me file does not work.

### Description

The example does not even compile. I could make it compile by changing the Delete part as follows
[ surrealdb.Delete returns two fields so the first has to be ignored]


```go
	if _, err = surrealdb.Delete[models.RecordID](db, *person2.ID); err != nil {
		panic(err)
	}

	// Delete all entries
	if _, err = surrealdb.Delete[models.Table](db, models.Table("persons")); err != nil {
		panic(err)
	}
```

But then I've got an error message, which I don't know how to fix 

```go
panic: cbor: cannot unmarshal map into Go struct field connection.RPCResponse[github.com/surrealdb/surrealdb.go/pkg/models.RecordID].result of type []interface {}

goroutine 1 [running]:
main.main()
	/home/pablo/work/prueba_surrealdb_go/ejemplo_basico/main.go:89 +0x691
``o

The are some opened issues for the example (like  #123 and #126). But I am not sure if they are the same issue.

### Is there an existing issue for this?

- [x] I have searched the existing issues

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

--- a/README.md
+++ b/README.md
@@ -138,12 +138,12 @@ func main() {
 	fmt.Printf("Selected all in persons table: %+v\n", persons)
 
 	// Delete an entry by ID
-	if err = surrealdb.Delete[models.RecordID](db, *person2.ID); err != nil {
+	if err = surrealdb.Delete[Person](db, *person2.ID); err != nil {
 		panic(err)
 	}
 
 	// Delete all entries
-	if err = surrealdb.Delete[models.Table](db, models.Table("persons")); err != nil {
+	if err = surrealdb.Delete[[]Person]](db, models.Table("persons")); err != nil {
 		panic(err)
 	}
 

